rapid7/custom_device_context.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import argparse
+import csv
+import requests
+
+from requests.auth import HTTPBasicAuth
+from typing import Dict, List
+
+SOURCE = "rapid7"
+
+# User can update values
+REPORT_ID = "Scan report"
+REPORT_TIMESTAMP = "2021-08-26T00:58:58Z"
+
+# Added default value to run API but user should map appropriate mac address
+MAC_ADDRESS = "98:01:a7:c7:5c:f8"
```

Comment:
On 2nd thoughts, let's not add any MAC address, otherwise it seems that the script can be left as is and will work. Let's add some comments about IP-> MAC mapping requirements.